File: adnetworks/Afio/MovieInterstitial6010.m
//
//  MovieInterstitial6010.m
//  MovieRewardSampleDev
//
//  Created by Zheng Gong on 2018/7/13.
//  Copyright © 2018年 A .D F. U. L. L. Y Co., Ltd. All rights reserved.
//

#import "MovieInterstitial6010.h"

@implementation MovieInterstitial6010

-(id)init {
    self = [super init];
    if (self) {
        [self setCancellable];
    }
    return self;
}

-(void)setCancellable {
    [super setCancellable];
    
    if (self.amoadInterstitialVideo) {
        [self.amoadInterstitialVideo setCancellable:YES];
    }
}

@end

File: adnetworks/Afio/MovieInterstitial6010.swift
//
//  MovieInterstitial6010.swift
//  MovieRewardTestApp
//
//  Created by Sungil Kim on 2020/05/20.
//  Copyright © 2020 Glossom, Inc. All rights reserved.
//

import Foundation
import ADFMovieReward

@objc(MovieInterstitial6010)

class MovieInterstitial6010: MovieReward6010 {
    override init() {
        super.init()
        setCancellable()
    }

    override func setCancellable() {
        amoadInterstitialVideo?.isCancellable = true
    }
}

@objc(MovieInterstitial6180)
class MovieInterstitial6180: MovieInterstitial6010 {
}

@objc(MovieInterstitial6181)
class MovieInterstitial6181: MovieInterstitial6010 {
}

@objc(MovieInterstitial6182)
class MovieInterstitial6182: MovieInterstitial6010 {
}

@objc(MovieInterstitial6183)
class MovieInterstitial6183: MovieInterstitial6010 {
}

@objc(MovieInterstitial6184)
class MovieInterstitial6184: MovieInterstitial6010 {
}
